### PR TITLE
[10.0] help_online fix export/import

### PR DESCRIPTION
**Pull request by lmignon**

### CODE REVIEW

**Comment by lmignon** [comment]
ping @Olivier-LAURENT @lasley @JonathanNEMRY

**Review by JonathanNEMRY — APPROVED** [review]

File: help_online/tests/common.py [hunk 1]
```diff
@@ -81,6 +81,7 @@ def createPage(self, pageName, imgXmlId=False):
             'type': 'qweb',
             'arch': arch,
             'page': True,
+            'key': 'website.' + pageName
```
Comment (lasley) [inline help_online/tests/common.py]: `'website.%s' % pageName` in order to handle type conversions

**Review by lasley — APPROVED** [review]
@lmignon one minor note & can you also please raise the module up by a patch version in the manifest?

**Review by adrienpeiffer — APPROVED** [review]
:+1:

**Review by moylop260 — APPROVED** [review]

**Review by felixvillafranca — APPROVED** [review]
Works

**Comment by dreispt** [comment]
It looks like this module is breaking the build:
https://travis-ci.org/OCA/web/jobs/237945481#L668

**Comment by lmignon** [comment]
@dreispt 
It looks like a commit has been added to the initial PR just before the merge and squashed into mines.... :rage: 
https://github.com/OCA/web/pull/584/commits/10608205082b99dae9a6cd775505efc4e515cd0b

**Comment by lmignon** [comment]
moreover, my commit history was clean and should not have been squashed. I don't want to be credited for this :bomb: :boom: commit. Commits from different authors can't be squashed!!!!!

**Comment by lmignon** [comment]
@Yajo Can you fix your mistake PLZ

**Comment by yajo** [comment]
Yes, @lmignon sorry, will fix ASAP.

**Pull request merged**